ops/hookcmds/_types.py

```diff
@@ -54,16 +64,14 @@ class Address:
     @classmethod
     def _from_dict(cls, d: AddressDict) -> Address:
         return cls(
-            hostname=d['hostname'],
-            value=d['value'],
-            cidr=d['cidr'],
+            hostname=d.get('hostname', ''),
+            value=d.get('value', ''),
+            cidr=d.get('cidr', ''),
         )
 
-
-BindAddressDict = TypedDict(
-    '_BindAddressDict',
-    {'mac-address': str, 'interface-name': str, 'addresses': list[dict[str, str]]},
-)
+    @property
+    def address(self) -> str:
```

Comment:
Why is this alias being added? It seems to me we should we call it either `value` or `address` (the latter seems clearer?) and avoid having an alias for new code/types.

Reply:
Yes, that was the wrong fix, it was meant to be in the loading. Changed.